.github/workflows/api-docs-repo.yaml

```diff
@@ -11,6 +11,10 @@ on:
 jobs:
   gen-crd-docs:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: read
+      pull-requests: write
```

Comment:
I don't think `pull-requests: write` will work.

Reply:
Ah it's using the PAT to create the PR so it's not needed too.

Reply:
Fixed